src/sentry/static/sentry/app/views/projectInstall/overview.jsx

```diff
@@ -32,14 +32,15 @@ class ProjectInstallOverview extends AsyncComponent {
 
   redirectToDocs = platform => {
     const {orgId, projectId} = this.props.params;
-    const prefix = recreateRoute('', {...this.props, stepBack: -3});
-    let rootUrl = `${prefix}install`;
 
-    if (this.isGettingStarted) {
-      rootUrl = `/organizations/${orgId}/projects/${projectId}/getting-started`;
-    }
+    const installUrl = this.isGettingStarted
+      ? `/organizations/${orgId}/projects/${projectId}/getting-started/${platform}`
+      : recreateRoute(`install/${platform}`, {
```

Comment:
Should these have trailing slashes?

```suggestion
      ? `/organizations/${orgId}/projects/${projectId}/getting-started/${platform}/`
      : recreateRoute(`install/${platform}/`, {
```